arayabrain/barebone-studio
# Refactoring loading icon display at API request

- Modifications
  - Reorganization of existing code
    - Unified variable names
    - Unification of <Loading /> component coding format
  - Add new loading icon display
    - Workflow screen
      - Initial display
    - Record screen
      - get experiments
      - delete experiment

- Test case
  - Check the loading icon on the following screen
  - standalone mode
    - Workflow screen
      - Initial display
    - Record screen
      - get experiments
      - delete experiment
    - Visualize screen
      - show ROI
  - multiuser mode
    - Add the following to the standalone mode case
    - Workspace screen
    - Account screen
    - AccountManager screen

- Test Result
  - standalone mode

[standalone-loading-icon.webm](https://github.com/arayabrain/barebone-studio/assets/122338895/be8ba254-069b-4a90-bb56-23ad104e9c25)

  - multiuser mode

[multiuser-loading-icon.webm](https://github.com/arayabrain/barebone-studio/assets/122338895/6166bc20-b087-47a7-87ca-5c33bc6f8264)
